cmd/xl-v1.go

```diff
@@ -145,9 +142,9 @@ func newXLObjects(storageDisks []StorageAPI) (ObjectLayer, error) {
 	}
 
 	// Figure out read and write quorum based on number of storage disks.
-	// READ and WRITE quorum is always set to (N/2) number of disks.
-	xl.readQuorum = readQuorum
-	xl.writeQuorum = writeQuorum
+	// READ Quorum is always set to (N/2) and WRITE quorum is (N/2 +1). N is the number of disks for metadata files.
+	xl.metaReadQuorum = readQuorum
+	xl.metaWriteQuorum = writeQuorum
```

Comment:
Move lines up into construction of xlObjects (lines 111 - 133)?

Reply:
@harshavardhana Can we get rid of defining metaWriteQuorum separately? 
And instead define a contant lie "WriteOverReadQuorum" to derive it from a read quorum? Then we can use the same constant in `storage-class.go`, line 212

By default it would be 1, but in certain cases increasing it can give additional Put security.

Reply:
Yeah I was thinking we don't need metaWriteQuorum since there is no reason for these variables.